src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs

```diff
@@ -149,21 +152,33 @@ public async Task<bool> CheckIfBlobExistsAsync(string blobPath)
             return await blob.ExistsAsync();
         }
 
-        private string GetMimeMapping(string filePath)
+        private (string ContentType, string CacheControl) GetBlobPropertiesByExtensions(string filePath)
         {
             var mimeMappings = new Dictionary<string, string>()
             {
-                {".svg", "image/svg+xml"}
+                {".svg", "image/svg+xml"},
+                {".version", "text/plain" }
+            };
+
+            var cacheMappings = new Dictionary<string, string>()
+            {
+                {".svg", "no-cache"}
             };
 
             if (string.IsNullOrWhiteSpace(filePath))
             {
                 throw new ArgumentException("An attempt to get the MIME mapping of an empty path was made.");
             }
 
-            return mimeMappings.TryGetValue(Path.GetExtension(filePath).ToLower(), out string mime) ?
-                mime :
+            string contentType = mimeMappings.TryGetValue(Path.GetExtension(filePath).ToLower(), out string cttType) ?
```

Comment:
Use `Invariant` to be a little safer:

```suggestion
            string contentType = mimeMappings.TryGetValue(Path.GetExtension(filePath).ToLowerInvariant(), out string cttType) ?
```